tags/analys_func.go

```diff
@@ -0,0 +1,101 @@
+package tags
+
+import (
+	"reflect"
+
+	"github.com/goreflect/gostructor/infra"
+)
+
+/*
+GetFunctionTypes - return slice of functions which should configuring sourceStruct structure
+*/
+func GetFunctionTypes(sourceStruct interface{}) []infra.FuncType {
+	summirize := []int{}
+	value := reflect.Indirect(reflect.ValueOf(sourceStruct))
```

Comment:
It's exported function, so you should check input data, because you may get here not a pointer but direct value

Reply:
i have get direct value, i don't have pointer =)

Reply:
I want to get the value I don't need a pointer here. If you read the documentation, whatever I transmit, whether it is a value or a link, in the output I will get a value of type reflect.Value
code from value.go:
```golang
// Indirect returns the value that v points to.
// If v is a nil pointer, Indirect returns a zero Value.
// If v is not a pointer, Indirect returns v.
func Indirect(v Value) Value {
	if v.Kind() != Ptr {
		return v
	}
	return v.Elem()
}
```